### PR TITLE
Refactor record active date

### PR DESCRIPTION
**Pull request by pjsde**
Refactor recordActiveDate() to avoid updating the 'updated_at' column.
Fix https://github.com/codeigniter4/shield/discussions/388

### CODE REVIEW

**Review by MGatner — APPROVED** [review]
CS Fixer error is unrelated to this PR - probably changes due to a new minor version of the tool. Thanks @pjsde, this looks good to me! See if @kenjis has any final thoughts.

File: src/Models/UserModel.php [hunk 1]
```diff
@@ -320,4 +320,14 @@ protected function saveEmailIdentity(array $data): array
 
         return $data;
     }
+
+    /**
+     * Updates the user's last active date.
+     */
+    public function saveActiveDate(User $user): void
```
Comment (kenjis) [inline src/Models/UserModel.php:327]: `updateActiveDate()` is better?
Because it is just updating only.
*save* implies insert or update.

Comment (pjsde) [reply src/Models/UserModel.php]: I agree. It's done. I also added one more test that was missing for the `AccessTokens` authenticator

**Review by kenjis — APPROVED** [review]

File: src/Models/UserModel.php (hunk 1 above)
Comment (kenjis) [reply src/Models/UserModel.php]: Thank you!

**Comment by MGatner** [comment]
Thanks all; glad to see this one in.

**Pull request merged**